H4API/lib/h4api/H4Base/H4Base.cpp

```diff
@@ -96,12 +97,12 @@ namespace h4
 
 	_H4API_ BOOL8 H4Version::game() const
 	{
-		return m_version == GameVersion::GAME;
+		return m_version == GameVersion::GAME || m_version == GameVersion::UNPATCHED_GAME;
```

Comment:
I'd be inclined of doing a switcharoo here: define the original entry point as GAME/EDITOR and have the modified ones be written as LEGACY_X or something to that effect. Thoughts?

Reply:
Makes sense if we assume that one standardized exe will be used to run mods.

Reply:
Agree, do you have time to make the modifications? If not I can do it this weekend.

Reply:
Changed GAME to LEGACY_PATCHED_GAME, EDITOR to LEGACY_PATCHED_EDITOR, UNPATCHED_GAME and UNPATCHED_EDITOR just to GAME and EDITOR

Reply:
Great, many thanks!